### PR TITLE
 Refactoring Simple Employee Document DTO

### PR DESCRIPTION
**Pull request by KAYEMBAJONATHAN**
> Add refactor Simple Employee Document DTO
SimpleEDto

image

### CODE REVIEW

File: RR.App/Program.cs [hunk 1]
```diff
@@ -10,6 +10,7 @@
 using HRIS.Models;
 using HRIS.Services.Services;
 using RabbitMQ.Client;
+using System.Xml.Linq;
```
Comment (tbddorothy) [inline RR.App/Program.cs:13]: Why did you add Linq?

**Review by AWallsRR — DISMISSED** [review]

**Review by Juanro-Minne — DISMISSED** [review]

**Comment by tbddorothy** [comment]
You have conflicts

**Comment by KAYEMBAJONATHAN** [comment]
> You have conflicts

@tbddorothy   resoved

**Review by Juanro-Minne — APPROVED** [review]

**Review by AWallsRR — APPROVED** [review]

**Review by ReinoRabbit — APPROVED** [review]
